test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageCreateModelExpressionTest.cs

```diff
@@ -148,7 +148,7 @@ public static TheoryData NotQuiteIdentityExpressions
             var page = CreatePage(viewContext);
 
             // Act
-            var result = page.CreateModelExpression(expression);
+            var result = page.CreateModelExpression<int>(expression);
```

Comment:
Must _not_ require explicit type parameters. This doesn't match the generated code.